### PR TITLE
Add Bengali language assets

### PR DESCRIPTION
**Pull request by Ladsgroup**
Tests fail for strange reasons that is beyond my knowledge

Bug: https://phabricator.wikimedia.org/T162620

### CODE REVIEW

**Comment by Ladsgroup** [comment]
Also, needs https://github.com/travis-ci/apt-package-whitelist/pull/4107/files to be merged first

**Comment by halfak** [comment]
We might need to disable that test.  It could be that the package is not available in our test vms.  See had to do the same for hindi and tamil.  See https://github.com/wiki-ai/revscoring/blob/master/.travis.yml#L51

**Comment by Ladsgroup** [comment]
Done

File: revscoring/languages/tests/util.py [hunk 1]
```diff
@@ -6,6 +6,7 @@ def compare_extraction(extractor, examples, counter_examples,
 
     for example in examples:
         wrapped = lwrap + example + rwrap
+        print(extractor.process(wrapped))
```
Comment (halfak) [inline revscoring/languages/tests/util.py]: We should probably drop this line.

**Comment by halfak** [comment]
Needs an update to requirements.txt for deltas 0.4.6

**Pull request merged**